### DISCUSSION TITLE
Build Instruction

### DISCUSSION THREAD

**Discussion by ceewanna** [Motion-Project/motionplus#25] [General]
I wonder if there is any build instruction for motionplus. I want to try it on my system. Currently I am exploring motion.

**Comment by Mr-Dave** [comment]
[The instructions are virtually identical to Motion.  A deb building script has now been put into the packaging project similar to the one used for Motion.  Specifics for MotionPlus are in the [motionplus_build](https://github.com/Motion-Project/motionplus/blob/master/doc/motionplus_build.html) document

**Reply by ceewanna** [reply]
Thanks. I wish there is build command for arch/manjaro linux as well.

**Reply by ceewanna** [reply]
I have built a motionplus with the following. Maybe it could be useful for someone using arch/manjaro.

sudo pacman -Sy base-devel
cd ~
git clone https://github.com/Motion-Project/motionplus.git
cd motionplus
autoreconf -fiv
./configure
make
sudo make install
